### ISSUE TITLE
Environment variables

### ISSUE THREAD

**Issue by Dzo67** [Mgiepz/saiku-adhoc#180]
Is it possible or will it become so
to use environment variables of the Pentaho BI Server in the query,
eg to let a user only see his own data?

The Pentaho Report Designer has a few predefined variables, as env::username, env::roles, env::roles-array etc. It would be handy to have them in Saiku ad hoc as well.

**Comment by Mgiepz** [comment]
No, there is no plan to do that. I think this is something that should 
be adressed in the metadata-layer, not in the report.

Am 19.02.2013 10:58, schrieb Dzo67:
